java/arcs/core/data/util/BUILD

```diff
@@ -8,7 +8,7 @@ licenses(["notice"])
 package(default_visibility = ["//visibility:public"])
 
 arcs_kt_library(
-    name = "util",
+    name = "data-util",
```

Comment:
Why this change?

Reply:
Apparently Kotlin/JS doesn't appreciate it when there are two build targets as deps with the same "name". Maybe we could instead modify the build macro to include the entire package path in the javascript lib filename?

Reply:
Ooof, that sounds incredibly annoying. Might need to look into that.